lib/ts/framework/loopback/framework.ts

```diff
@@ -92,7 +92,7 @@ export class LoopbackResponse extends BaseResponse {
     sendHTMLResponse = (html: string) => {
         if (!this.response.writableEnded) {
             this.response.set("Content-Type", "text/html");
-            this.response.status(this.statusCode).send(Buffer.from(html));
+            this.response.status(this.statusCode).send(getBuffer().from(html));
```

Comment:
Please check that we actually need to make this into a buffer.
If we do, I'd prefer if we used a `getBufferFromString` function instead of `getBuffer().from`

Reply:
> Please check that we actually need to make this into a buffer.

I don't think it's necessary as the expected type of `html` is string and express takes care of it automatically.